1. Python 3.13.0 will lead to error: ** No module named 'pydantic_core._pydantic_core'
change back to Python 3.12.3 no problem.
2. 
